internal/output/output.go

```diff
@@ -38,7 +38,12 @@ func Process(head *models.Header, funcs []*models.Function, opt *Options) ([]byt
 	if err := writeTests(b, head, funcs, opt); err != nil {
 		return nil, err
 	}
-	out, err := imports.Process(tf.Name(), b.Bytes(), nil)
+
+	return ProcessImports(tf.Name(), b.Bytes())
+}
+
+func ProcessImports(filename string, src []byte) ([]byte, error) {
```

Comment:
According to https://golang.org/doc/effective_go.html#commentary:

> Every exported (capitalized) name in a program should have a doc comment.

It's helpful to document what `ProcessImports` does (including details about its parameters, what happens when `src` is nil, etc.). Without documentation, it's harder to know what its responsibility and expected behavior is.